firefox-ios/Extensions/NotificationService/NotificationService.swift

```diff
@@ -30,7 +39,12 @@ class NotificationService: UNNotificationServiceExtension {
 
         let userInfo = request.content.userInfo
 
-        let content = request.content.mutableCopy() as! UNMutableNotificationContent
+        guard let content = request.content.mutableCopy() as? UNMutableNotificationContent else {
+            logger.log("Unable to create a mutable copy of UNNotificationContent from the notification request",
```

Comment:
We should call `contentHandler(request.content)` here instead of the log so the notification doesn't get deferred to the process termination.